exotica/src/KinematicTree.cpp

```diff
@@ -311,11 +310,8 @@ void KinematicTree::resetModel()
     // Remove all CollisionShapes
     if (Server::isRos())
     {
-        visualization_msgs::MarkerArray msg;
-        visualization_msgs::Marker mrk;
-        mrk.action = 3;  // visualization_msgs::Marker::DELETEALL; // NB: enum only defined in ROS-J and newer, functionality still there
-        msg.markers.push_back(mrk);
-        shapes_pub_.publish(msg);
+        msg_marker.markers.emplace_back();
+        msg_marker.markers.end()->action = visualization_msgs::Marker::DELETEALL;
```

Comment:
Replace with `action = 3;  // visualization_msgs::Marker::DELETEALL; // NB: enum only defined in ROS-J and newer, functionality still there` to avoid breaking indigo support (it's the same).